modules/sdk-coin-hash/src/lib/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const validDenoms = ['nhash', 'uhash', 'mhash', 'hash'];
+export const accountAddressRegex = /^(tp)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
+export const validatorAddressRegex = /^(tpvaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
```

Comment:
source?

Reply:
@DinshawKothari 
Prefix of validator address is different for test and prod.
Please have a look the validator address
Test: https://explorer.test.provenance.io/validators
Prod: https://explorer.provenance.io/validators